Add release notes and document env var for Enterprise 3.7 (#2042)
sensu/sensu-docs: content/sensu-enterprise/3.7/changelog.md

- [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
- [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)

## Enterprise 3.7.0 Release Notes {#enterprise-v3-7-0}

> **December 19, 2019** &mdash; Sensu Enterprise version 3.7.0 has been

Reviewer: ```suggestion
**January 7, 2020** &mdash; Sensu Enterprise version 3.7.0 has been
```

released and is available for immediate download. Please note the
following improvements:

### CHANGES {#enterprise-v3-7-0-changes}

Built on [Sensu Core 1.9.0][core-v1-9-0]:

> - **IMPROVEMENT**: You can now configure the threshold for Java CMSInitiatingOccupancyFraction garbage collection strategy via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).

Reviewer: ```suggestion
- **IMPROVEMENT**: You can now configure the Initiating Occupancy Fraction threshold for Java garbage collection via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).
```

- [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
- [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)

## Enterprise 3.7.0 Release Notes {#enterprise-v3-7-0}

**December 19, 2019** &mdash; Sensu Enterprise version 3.7.0 has been
released and is available for immediate download. Please note the
following improvements:

### CHANGES {#enterprise-v3-7-0-changes}

> 

Author: ```suggestion
```

- [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
- [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)

## Enterprise 3.7.0 Release Notes {#enterprise-v3-7-0}

**December 19, 2019** &mdash; Sensu Enterprise version 3.7.0 has been
released and is available for immediate download. Please note the
following improvements:

### CHANGES {#enterprise-v3-7-0-changes}

> - **BUGFIX**: Fixed Travis CI builds.

Author: ```suggestion
```

Built on [Sensu Core 1.9.0][core-v1-9-0]:

> - **IMPROVEMENT**: You can now tune CMSInitiatingOccupancyFraction via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).

Reviewer: ```suggestion
- **IMPROVEMENT**: You can now configure the threshold for Java CMSInitiatingOccupancyFraction garbage collection strategy via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).
```
